web/html/src/components/panels/BootstrapPanel.tsx

```diff
@@ -31,6 +32,4 @@ BootstrapPanel.defaultProps = {
   footer: undefined,
 };
```

Comment:
Above: I think `defaultProps` is no-op here, right?

Reply:
Good catch, that's indeed the case. Thanks for looking through it. 👍